tardis/visualization/tools/rpacket_visualization.py

```diff
@@ -0,0 +1,691 @@
+import astropy.units as u
+import plotly.express as px
+import plotly.graph_objects as go
+import math
+import pandas as pd
+import numpy as np
+import random
+import logging
```

Comment:
Not a strict rule but generally we import in following order:
- inbuilt modules
- numpy, pandas
- other dependencies (astropy, plotly, etc.)

See other modules